firebase-firestore/src/test/java/com/google/firebase/firestore/util/MapperTest.java

```diff
@@ -709,7 +709,10 @@ public static void setValue1(String value1) {
 
   private enum ComplexEnum {
     One("one"),
-    Two("two");
+    Two("two"),
+
+    @PropertyName("three")
+    THREE("three");
```

Comment:
Please make either the PropertyName or the value different here so there's no chance the test is accidentally succeeding.

Reply:
I made this change.. PropertyName is Three and value is three